runtime/vm/mz64/UpcallThunkGen.cpp

```diff
@@ -287,6 +287,7 @@ createUpcallThunk(J9UpcallMetaData *metaData)
 	}
 	for (I_32 i = 0; i < lastSigIdx; i++) {
 		U_32 argSizeInBytes = sigArray[i].sizeInByte;
+		int totalStackSlotsReq = ((argSizeInBytes - 1) / 8) + 1;
```

Comment:
Given that we are not going to change this variable in the code, can we use `const int`?

Reply:
This should be addressed in https://github.com/eclipse-openj9/openj9/commit/4632a1437a32b27e8d036445b96bf05668838f6f now.